### PR TITLE
Localize `N/A`

### PR DESCRIPTION
**Pull request by yardenshoham**
I saw this string hardcoded in a few places

### CODE REVIEW

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -23,6 +23,7 @@ enable_javascript = This website requires JavaScript.
 toc = Table of Contents
 licenses = Licenses
 return_to_gitea = Return to Gitea
+short_not_available = N/A
```
Comment (wxiaoguang) [inline options/locale/locale_en-US.ini]: `abbr_not_available` ?

**Review by wxiaoguang — APPROVED** [review]

**Comment by silverwind** [comment]
Hmm, I do wonder what happens when translators introduce longer strings for it. Can the places where it's used realistically render larger content?

**Comment by yardenshoham** [comment]
Yes
image

**Comment by silverwind** [comment]
Hmm, I doubt we will get many sensical translations. [N/A](https://en.wikipedia.org/wiki/N/A) is a english-only abbreviation and I wouldn't know how to translate it into german for example, as the language has no equivalent.

How about just rendering a neutral `-`. That will work fine as well in these tables and needs no translation.

**Comment by wxiaoguang** [comment]
I also agree to keep the "N/A", it is a well-known word, especially in technical field.

Either is fine.

**Comment by wxiaoguang** [comment]
Actually, I think some N/A strings could be replaced by some more accurate word.

The last time I used "N/A" because there is no result. But I really do not want to put another "No record" translation there ....

(well, I admit that I am somewhat lazy on some problems ....)


While from another perspective, maybe `N/A` is not that bad, I am pretty neutral.

**Comment by silverwind** [comment]
Let's just use `-`. It's already in use in the same exact lists, needs no translation and is perfectly short. It's a perfect fit.

**Comment by silverwind** [comment]
> Let's just use `-`. It's already in use in the same exact lists, needs no translation and is perfectly short. It's a great fit.
> 
> @yardenshoham why close? Not interested in continuing? Should I raise another?

https://github.com/go-gitea/gitea/pull/24474

**Pull request closed without merging**